extract_endpoint/tests/test_post_to_endpoint.py

```diff
@@ -0,0 +1,127 @@
+import io
+import subprocess
+import typing
+from http import HTTPStatus
+import psutil
+import pytest
+import _pytest
+import requests
+from azure.storage import blob
+from extract_endpoint import post_to_endpoint, azure_utils
+
+
+class NamedStream(io.BytesIO):
+    name: str
```

Comment:
This was required to deal with a python bug that was just recently fixed: https://github.com/python/typeshed/issues/1790

Reply:
That is weird and annoying

Reply:
:( indeed. At least it'll be fixed in 3.7